app/schemas.py

```diff
@@ -62,7 +63,7 @@ def _validate_datetime_not_in_past(dte, msg="Date cannot be in the past"):
         raise ValidationError(msg)
 
 
-class BaseSchema(ma.SQLAlchemyAutoSchema):
+class BaseSchema(ma.SQLAlchemyAutoSchema):  # type: ignore
```

Comment:
Should SQLAlchemyAutoScheme be imported as well like Schema ?

Reply:
I tried that but it caused a lot of tests to fail. I think we are initiallizing some things on our instance of Marshmallow that need to be available in these classes.